shell/src/main/java/org/apache/accumulo/shell/commands/GrepCommand.java

```diff
@@ -114,8 +144,11 @@ public Options getOptions() {
     final Options opts = super.getOptions();
     numThreadsOpt = new Option("nt", "num-threads", true, "number of threads to use");
     negateOpt = new Option("v", "negate", false, "only include rows without search term");
+    profileNameOpt = new Option("pn", "profile-name", true,
```

Comment:
Not completely sure, but I think the earlier call to `super.getOptions();` may already add a profile option.

Reply:
agreed, code updated